relnotes/v0.38.0.md

```diff
@@ -10,7 +10,7 @@
 
 ### Bug fixes
 
-* Add `require 'time'` to `remote_config.rb` to avoid "undifined method `rfc2822'". ([@necojackarc][])
+* Add `require 'time'` to `remote_config.rb` to avoid "undifined method \`rfc2822'". ([@necojackarc][])
```

Comment:
Actually I think rfc2822 should be in simple backticks and the rest was a mistake.